en/SUMMARY.md

```diff
@@ -76,6 +76,9 @@
   * [FrSky Telemetry](peripherals/frsky_telemetry.md)
   * [RTK GPS - Here+](peripherals/rtk_gps_hex_hereplus.md)
   * [RTK GPS - Drotek XL](peripherals/rtk_gps_drotek_xl.md)
+  * [Telemetry](telemetry/README.md)
```

Comment:
Should we rename this to `MAVLink Telemetry`, to make it clear it's separate from `FrSky Telemetry`?

Reply:
The complication is that the radios might also be used for other protocols and data. How about Telemetry Radios - differentiates a little by making it clear that these are radios that are just for telemetry.